src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -0,0 +1,22 @@
+package ca.bc.gov.open.pssg.rsbc.dps.dpsvalidationservice;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GetValidOpenDFCMCase {
+
+    @RequestMapping(value = "/getValidOpenDFCMCase",
+            produces = {"application/xml"},
+            method = RequestMethod.GET)
+    public String getValidOpenDFCMCase(@RequestParam(value = "driversLicense", required = true) String driversLicense, @RequestParam(value = "surcode", required = true) String surcode) {
+        /* TO DO implement the logic as in the webmethods */
+        return "<GetValidOpenDFCMCase>\n" +
```

Comment:
Is this best practice for string concatenation? @alexjoybc I know we had issues with that in bc-paris, just want to make sure.